### PR TITLE
Multiple fixes to the PostgresProvider

### PR DESCRIPTION
**Pull request by belochub**
* Fix category retrieval by Id for get() and set() methods.
* Update SQL queries syntax.
* Ignore Id field when validating objects in the set() method.

### CODE REVIEW

File: lib/pg.provider.js [hunk 1]
```diff
@@ -166,8 +173,9 @@ class PostgresProvider extends StorageProvider {
         .filter(key => key !== 'Id');
       const values = fields.map(key => obj[key]);
       fields = fields.map(escapeIdentifier);
-      const setQuery = `UPDATE ${category} SET (${fields.join(', ')}) = ` +
-        `(${generateQueryParams(fields.length)})`;
+      const setQuery = `UPDATE ${escapeIdentifier(category)}` +
+        ` SET (${fields.join(', ')}) =` +
+        ` ROW (${generateQueryParams(fields.length)})`;
```
Comment (tshemsedinov) [inline lib/pg.provider.js]: I know following syntax: `UPDATE table SET field1=value1, field1=value1 ... WHERE ...`

File: lib/pg.provider.js [hunk 2]
```diff
@@ -153,7 +157,10 @@ class PostgresProvider extends StorageProvider {
         return;
       }
 
-      const error = this.gs.schema.validateCategory(category, obj);
+      const toValidate = Object.assign({}, obj);
+      delete toValidate.Id;
```
Comment (lundibundi) [inline lib/pg.provider.js]: I find this strange and inefficient, we should introduce some sort of mechanism to remove the need for this.
/cc @tshemsedinov @nechaido @belochub

Comment (belochub) [reply lib/pg.provider.js]: @lundibundi I will add a `TODO` comment here for now.

**Comment by belochub** [comment]
Landed in https://github.com/metarhia/globalstorage/commit/18f5c5552dfaa4038c20a860c0bde2ce5040ecc2.

**Pull request closed without merging**